exporter/collector/integrationtest/protos/fixtures.pb.go

```diff
@@ -14,7 +14,7 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.36.2
+// 	protoc-gen-go v1.36.5
```

Comment:
You might need to update the version of protoc-gen-go in the tools directory to match this.  Is there a reason you want this updated?

Reply:
`protoc-gen-go` is installed by the Makefile, but does not appear to be in the `tools/go.mod` file. Shall I add it there? And should I just choose this version?

Reply:
Oh actually I misunderstood, it's just a sub-package of the `protobuf` package. I didn't actually want to do this update, so I can't tell why the `check-clean-work-tree` target is failing.

Reply:
Updating protobuf did end up being the solution. I'm not sure why this was necessary, don't fully understand how all these bits fit together